code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -579,6 +579,14 @@ Please contact me on #coderbus IRC. ~Carnie x
 	var/standing	= image("icon"=female_clothing_icons["[t_color]_s"], "layer"=-layer)
 	return(standing)
 
+/mob/living/carbon/human/proc/get_overlays_copy(var/inhands=0)

```

Comment:
Make this more generic please, instead of ```var/inhands=0``` have the argument be ```var/list/unwantedLayers```
and simply do ```if(i in unwantedLayers) continue```

I ask you to make this more generic as it's really not needed, 90% of stuff to do with copying overlays is as simple as
```X.overlays = Y.overlays.Copy()``` A more generic version of your proc WOULD have usage of it's own.